account_reconcile_model_strict_match_amount/tests/test_account_reconcile_model_strict_match_amount.py

```diff
@@ -15,6 +15,10 @@ def setUpClass(cls):
         cls.partner_6 = cls.env["res.partner"].create({"name": "partner_6"})
         cls.invoice_line_5 = cls._create_invoice_line(150, cls.partner_3, "out_invoice")
         cls.invoice_line_5.ref = "ABC001XYZ"
```

Comment:
```suggestion
```

Reply:
Thanks for updating. How about removing this?

Reply:
@grindtildeath will you handle this comment?